### ISSUE TITLE
Refactor new_initial_negatives into a function

### ISSUE THREAD

**Issue by pbchase** [ctsit/tt_engine#38]
Refactor `new_initial_negatives` into a function. `write_lab_results_to_results_project.R` is too big. We need to split it into components with minimal interdependency to make it easier to maintain. please discuss this with @pbchase to expand on this issue.

Do not start this until PR #37 is merged.

**Comment by pbchase** [comment]
The content in question starts at https://github.com/ctsit/tt_engine/blob/4754080628c5472712ba2b7a25834c538184598b/write_lab_results_to_results_project.R#L104 and runs through the end of the script.

This code uses these inputs: `raw_lab_result_data`, `aligned_lab_result_ids`, `all_rows_in_redcap_results`, `script_run_time`

Make these changes:

1. Refactor L104 through the end of the script into a function.
1. After the refactor commit, change this function to use the UFID from `align_lab_results_to_person_data` as the authoritative UFID.
1. Also consider the effect of replacing the left join with an inner join at https://github.com/ctsit/tt_engine/blob/4754080628c5472712ba2b7a25834c538184598b/write_lab_results_to_results_project.R#L121. It should probably be an inner join

**Comment by pbchase** [comment]
addressed by PR #43
